src/acp/acp.test.ts

```diff
@@ -58,6 +57,15 @@ jest.mock("../fetcher.ts", () => ({
   ),
 }));
 
+function mockResponseFrom(
+  body?: BodyInit | null,
+  init?: ResponseInit,
+  url?: string
+): Response {
+  const response = new Response(body, init);
+  jest.spyOn(response, "url", "get").mockReturnValue(url ?? "");
+  return response;
+}
```

Comment:
Why can't the `mockResponse` from `../test.internal.ts` be used. I'm also still unconvinced that this is required at all; shouldn't `new Response(body, init)` suffices since I can't see any tests against the spyed on object.

Reply:
It absolutely can, I just decided to have `mockResponse` in a shared file after I wrote this and forgot to come back and use it here. Fixed in d48e1dff.